rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPEntityMapper.scala

```diff
@@ -314,11 +315,33 @@ class LDAPEntityMapper(
           )
         }
       }
+      // custom properties mapped as NodeProperties
+      if(node.id.value == "root") {
+        println(inventoryEntry)
+      }
+      val customProperties = inventoryEntry.valuesFor(A_CUSTOM_PROPERTY).flatMap { json =>
+        import inventoryMapper.CustomPropertiesSerialization._
+        println("*** custom property:" + json)
+        json.toCustomProperty match {
+          case Left(ex)  =>
+            logger.error(s"Error when trying to deserialize Node Custom Property, it will be ignored: ${ex.getMessage}", ex)
+            None
+          case Right(cs) =>
+            Some(NodeProperty(cs.name, cs.value, Some(NodeProperty.customPropertyProvider)))
+        }
+      }
+      if(customProperties.nonEmpty) {
+        println("*"*30+customProperties)
+      }
+      val customPropertyNames = customProperties.map( _.name ).toSet
 
       // fetch the inventory datetime of the object
       val dateTime = inventoryEntry.getAsGTime(A_INVENTORY_DATE) map(_.dateTime) getOrElse(DateTime.now)
       NodeInfo(
-          node
+          node.copy(properties =
+            // filter out properties overriden by inventory property
+            node.properties.filterNot(x => customPropertyNames.contains(x.name) ) ++ customProperties
```

Comment:
i'm not sure I get the expected behaviour there ; does it completly replaces node properties from, say, datasource ?

Reply:
it would, but I'm not sure it's what we want. Perhaps I could only override properties for default provider. 
And in all cases, I should add a log "warn" (or info?) to tell that a property is overriden

Reply:
ok, I was under the impression that we would keep the different type of properties (those from datasources, those from the user, and those from the node), with the possibility to use either of them (or to have a clear explaination of what wins in each case, but still show all the data)